src/default-modules/ui/bottom-block/bottom-block.ts

```diff
@@ -139,6 +139,116 @@ export default class BottomBlock {
     this.view.hideLogo();
   }
 
+  /**
+   * Method for showing logo.
+   * @example
+   * player.showLogo();
+   */
+  @playerAPI()
+  showLogo() {
+    this.view.showLogo();
+  }
+
+  /**
+   * Method for showing play control.
+   * @example
+   * player.showPlayControl();
+   */
+  @playerAPI()
+  showPlayControl() {
+    this.view.showPlayControl();
+  }
```

Comment:
how do we control method ordering in the doc, according to code? it will be cool to have show/hide methods together, like `showPlayControl`, `hidePlayControl`, `showVolumeControl`, `hideVolumeControl`, etc

Reply:
in the doc methods ordered in the same order like in code.
If we need some other order, need to decide what criteria should be used for ordering.